models/Info.js

```diff
@@ -0,0 +1,55 @@
+class Info {
+  constructor() {
+    this.name = ''
+    this.email = ''
+    this.phone = ''
+    this.selectedPlan = ['Arcade', 'Advance', 'Pro']
+    this.frequency = ['Monthly', 'Yearly']
+    this.addOns = ['Online service', 'Larger storage', 'Customizable profile']
+  }
+
+  // Methods to update data for each section of the form
+  updateName(name) {
```

Comment:
in other programming languages, the convention is tu use `setFieldName` instead of `updateFieldName`